src/global/signal_handler.cc

```diff
@@ -173,6 +173,9 @@ static void handle_fatal_signal(int signum)
   BackTrace bt(1);
   ostringstream oss;
   bt.print(oss);
+  if (signum == SIGSEGV) {
+    oss << std::endl << "Fault at location: " << siginfo->si_addr << std::endl;
```

Comment:
While you're here, you might as well dump the whole struct. It's all useful information.